Outer loop Ubuntu legs pass all tests but then fail with IOException

http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/outerloop_ubuntu14.04_release/8/consoleFull
```
[xUnit] [ERROR] - IO error scanning directory '/mnt/j/workspace/dotnet_corefx/outerloop_ubuntu14.04_debug/bin/tests'
ERROR: Step ‘Publish xUnit test result report’ aborted due to exception: 
java.io.IOException: remote file operation failed: /mnt/j/workspace/dotnet_corefx/outerloop_ubuntu14.04_debug at hudson.remoting.Channel@79976537:Azure0227030641: hudson.util.IOException2: There are some problems during the conversion into JUnit reports: 
	at hudson.FilePath.act(FilePath.java:986)
	at hudson.FilePath.act(FilePath.java:968)
	at org.jenkinsci.plugins.xunit.XUnitProcessor.performTests(XUnitProcessor.java:145)
	at org.jenkinsci.plugins.xunit.XUnitProcessor.performXUnit(XUnitProcessor.java:88)
	at org.jenkinsci.plugins.xunit.XUnitPublisher.perform(XUnitPublisher.java:142)
	at org.jenkinsci.plugins.xunit.XUnitPublisher.perform(XUnitPublisher.java:134)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:782)
	at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:723)
	at hudson.model.Build$BuildExecution.post2(Build.java:185)
	at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:668)
	at hudson.model.Run.execute(Run.java:1763)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:410)
Caused by: hudson.util.IOException2: There are some problems during the conversion into JUnit reports: 
	at org.jenkinsci.plugins.xunit.service.XUnitTransformer.invoke(XUnitTransformer.java:183)
	at org.jenkinsci.plugins.xunit.service.XUnitTransformer.invoke(XUnitTransformer.java:44)
	at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2719)
	at hudson.remoting.UserRequest.perform(UserRequest.java:120)
	at hudson.remoting.UserRequest.perform(UserRequest.java:48)
	at hudson.remoting.Request$2.run(Request.java:326)
	at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
	at ......remote call to Azure0227030641(Native Method)
	at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1416)
	at hudson.remoting.UserResponse.retrieve(UserRequest.java:220)
	at hudson.remoting.Channel.call(Channel.java:781)
	at hudson.FilePath.act(FilePath.java:979)
	... 14 more
Caused by: IO error scanning directory '/mnt/j/workspace/dotnet_corefx/outerloop_ubuntu14.04_debug/bin/tests'
	at org.apache.tools.ant.DirectoryScanner.scandir(DirectoryScanner.java:1207)
	at org.apache.tools.ant.DirectoryScanner.checkIncludePatterns(DirectoryScanner.java:1030)
	at org.apache.tools.ant.DirectoryScanner.scan(DirectoryScanner.java:909)
	at org.apache.tools.ant.types.AbstractFileSet.getDirectoryScanner(AbstractFileSet.java:499)
	at org.apache.tools.ant.types.AbstractFileSet.getDirectoryScanner(AbstractFileSet.java:460)
	at org.jenkinsci.plugins.xunit.service.XUnitReportProcessorService.findReports(XUnitReportProcessorService.java:73)
	at org.jenkinsci.plugins.xunit.service.XUnitTransformer.invoke(XUnitTransformer.java:92)
	at org.jenkinsci.plugins.xunit.service.XUnitTransformer.invoke(XUnitTransformer.java:44)
	at hudson.FilePath$FileCallableWrapper.call(FilePath.java:2719)
	at hudson.remoting.UserRequest.perform(UserRequest.java:120)
	at hudson.remoting.UserRequest.perform(UserRequest.java:48)
	at hudson.remoting.Request$2.run(Request.java:326)
	at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:68)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[BFA] Scanning build for known causes...
..[BFA] Found failure cause(s):
[BFA] Hung processs on target machine from category Infrastructure
```

Also, why after the tests are run is it then doing a native build?

--- a/netci.groovy
+++ b/netci.groovy
@@ -308,6 +308,7 @@ branchList.each { branchName ->
                         --coreclr-bins \${WORKSPACE}/bin/Product/${osGroup}.x64.Release/ \\
                         --mscorlib-bins \${WORKSPACE}/bin/Product/${osGroup}.x64.Release/ \\
                         --outerloop
+                    sudo find . -name \"testResults.xml\" -exec chmod 777 {} \\;
                     """)
                 }
                 

